src/Deployer.sol

```diff
@@ -0,0 +1,130 @@
+// SPDX-License-Identifier: GPL-3.0-only
+pragma solidity ^0.8.17;
+
+import {AddressDriver} from "./AddressDriver.sol";
+import {Caller} from "./Caller.sol";
+import {DripsHub} from "./DripsHub.sol";
+import {Managed, ManagedProxy} from "./Managed.sol";
+import {NFTDriver} from "./NFTDriver.sol";
+import {ImmutableSplitsDriver} from "./ImmutableSplitsDriver.sol";
+
+contract Deployer {
+    // slither-disable-next-line immutable-states
+    address public creator;
+
+    DripsHub public dripsHub;
+    bytes public dripsHubArgs;
+    uint32 public dripsHubCycleSecs;
+    DripsHub public dripsHubLogic;
+    bytes public dripsHubLogicArgs;
```

Comment:
The deployments are internal transactions, so it's not easy to get their arguments. Foundry only fetches the whole transaction arguments, I think that even the raw Etherscan API doesn't expose data like this. This would be fine, but we need the arguments for contracts verification on Etherscan.

Reply:
We could reconstruct it in the bash script (since we know the params) but it might be cleaner this way.